recipes/htstream/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+mkdir build && cd build
+
+cmake -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON -DCMAKE_BUILD_TYPE=RELEASE -DCMAKE_OSX_DEPLOYMENT_TARGET=10.9 -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_LIBRARY_PATH=${PREFIX}/lib -DCMAKE_INCLUDE_PATH=${PREFIX}/include ..
```

Comment:
It's a minor thing, but you might want to split the `cmake` command up over multiple lines for readability.